cel/library.go

```diff
@@ -260,6 +260,27 @@ func (stdLibrary) ProgramOptions() []ProgramOption {
 // be expressed with `optMap`.
 //
 //	msg.?elements.optFlatMap(e, e[?0]) // return the first element if present.
+
+// # First
+//
+// Introduced in version: 2
+//
+// Returns an optional with the first value from the right hand list, or
+// optional.None.
+//
+// [1, 2, 3].first().value() == 1
+
+// # Last
+//
+// Introduced in version: 2
+//
+// Returns an optional with the last value from the right hand list, or
+// optional.None.
+//
+// [1, 2, 3].last().value() == 3
```

Comment:
Would you mind adding examples of empty lists for `first()` and `last()`? Also, don't forget to update the `README.md` since often folks look there for high-level docs within the extensions folder. Thanks!

Reply:
Yes, thank you for clarifying, the `ext/README.md` is the one I meant.